include/openenclave/bits/sgx/sgxproperties.h

```diff
@@ -34,13 +34,18 @@ typedef struct _oe_sgx_enclave_image_info_t
 #define OE_SGX_FLAGS_KSS 0x0000000000000080ULL
 #define OE_SGX_SIGSTRUCT_SIZE 1808
 
+typedef struct _oe_sgx_enclave_flags_t
+{
+    uint32_t capture_pf_gp_exceptions : 1;
+    uint32_t reserved : 31;
+} oe_sgx_enclave_flags_t;
```

Comment:
Out of curiosity, why do we need a separate structure instead of fields in the parent struct?

Reply:
The oe_sgx_enclave_flag_t is passed as an argument such that we can pass down the information to enclave signing functions.

Current behavior is
- the oe_sgx_enclave_flags_t struct is initialized based on the _OE_SET_ENCLAVE_SGX macro. The oesign tool can overwrite the struct based on the conf file. After that, the struct is passing down to `oe_sgx_sign_enclave_*` functions as an argument,